From 63414719dfa99f4edfb2d7f07617679f52636340 Mon Sep 17 00:00:00 2001
From: Paul McCaughtry <paul.mccaughtry@cloudera.com>
Date: Thu, 23 Oct 2014 07:32:06 -0500
Subject: [PATCH 0070/1173] HUE-2435 [oozie] ensure consistency of coordinator
 rerun actions bar with other dashboards in oozie

Updated filter input box to be same size as other search fields in Oozie
Added icon to Rerun button and disabled button when jobs are running
---
 .../dashboard/list_oozie_coordinator.mako          |   12 +++++-------
 apps/oozie/static/css/coordinator.css              |    4 ++++
 2 files changed, 9 insertions(+), 7 deletions(-)

diff --git a/apps/oozie/src/oozie/templates/dashboard/list_oozie_coordinator.mako b/apps/oozie/src/oozie/templates/dashboard/list_oozie_coordinator.mako
index 8194f44..701a51d 100644
--- a/apps/oozie/src/oozie/templates/dashboard/list_oozie_coordinator.mako
+++ b/apps/oozie/src/oozie/templates/dashboard/list_oozie_coordinator.mako
@@ -156,16 +156,14 @@ ${ layout.menubar(section='coordinators', dashboard=True) }
             <div class="tab-pane active" id="calendar">
               <div class="clearfix" style="margin-bottom: 1em;">
                 <div class="pull-left">
-                  <input type="text" data-bind="textInput: searchFilter, value: searchFilter,  valueUpdate: 'input'" class="input-large search-query" placeholder="${_('Filter results')}" style="margin-right: 1em; position: relative; top: -1px;">
+                  <input type="text" data-bind="textInput: searchFilter, value: searchFilter,  valueUpdate: 'input'" class="input-xlarge search-query" placeholder="${_('Filter results')}">
                   % if has_job_edition_permission(oozie_coordinator, user):
-                      <button data-bind="enable: selectedActions().length > 0" class="btn btn-primary rerun-btn
+                      <button data-bind="enable: selectedActions().length > 0" class="btn btn-primary rerun-btn action-button"
                          % if oozie_coordinator.is_running() or oozie_coordinator.status in ('KILLED', 'FAILED'):
-                           hide
+                           disabled="disabled"
                          % endif
-                      "
-                        data-rerun-url="${ url('oozie:rerun_oozie_coord', job_id=oozie_coordinator.id, app_path=oozie_coordinator.coordJobPath) }"
-                      style="margin-bottom: 5px">
-                        ${ _('Rerun') }
+                        data-rerun-url="${ url('oozie:rerun_oozie_coord', job_id=oozie_coordinator.id, app_path=oozie_coordinator.coordJobPath) }">
+                        <i class="fa fa-refresh"></i> ${ _('Rerun') }
                       </button>
                   % endif
                 </div>
diff --git a/apps/oozie/static/css/coordinator.css b/apps/oozie/static/css/coordinator.css
index 7171554..6b98e68 100644
--- a/apps/oozie/static/css/coordinator.css
+++ b/apps/oozie/static/css/coordinator.css
@@ -36,4 +36,8 @@
 
 .manage {
   padding: 0;
+}
+
+.search-query {
+   margin-right: .75em;
 }
\ No newline at end of file
-- 
1.7.9.5

